pkg/users/client.go

```diff
@@ -134,32 +130,39 @@ type ListUsersOpts struct {
 }
 
 type CreateUserOpts struct {
-	Email     string `json:"email"`
-	Password  string `json:"password"`
-	FirstName string `json:"first_name,omitempty"`
-	LastName  string `json:"last_name,omitempty"`
-
-	// EmailVerified states whether the user's email address was
-	// previously verified. You should normally use the Email Verification API
-	// to verify a user's email address
-	EmailVerified bool `json:"email_verified,omitempty"`
+	Email         string `json:"email"`
+	Password      string `json:"password"`
+	FirstName     string `json:"first_name,omitempty"`
+	LastName      string `json:"last_name,omitempty"`
+	EmailVerified bool   `json:"email_verified,omitempty"`
 }
 
-type AddUserToOrganizationOpts struct {
-	User         string `json:"id"`
-	Organization string `json:"organization_id"`
+type Session struct {
+	ID                        string                     `json:"id"`
+	Token                     string                     `json:"token"`
+	CreatedAt                 string                     `json:"created_at"`
+	ExpiresAt                 string                     `json:"expires_at"`
+	AuthorizedOrganizations   []Organization             `json:"authorized_organizations"`
```

Comment:
There should be an `AuthorizedOrganization` struct, which has just 1 field `organization` within it, which has name and ID. It looks a little extra but it's by design.

Reply:
Will make that update, thanks for confirming